docs/pipelines_user_guide.rst

```diff
@@ -0,0 +1,260 @@
+.. comment:
+    SPDX-FileCopyrightText: 2017-2022 Contributors to the OpenSTEF project <korte.termijn.prognoses@alliander.com>
+    SPDX-License-Identifier: MPL-2.0
+
+.. _pipeline_user_guide:
+
+Pipelines user guide
+=============================
+
+As mentioned in the :ref:`concepts <concepts>` section, tasks are an extension of pipelines, that include getting data from a database, 
+raising task exceptions, and writing data to a database. In the operational setting within Alliander, both tasks and pipelines are used
```

Comment:
Maybe nice to make this more general: "In the operational setting, both tasks and pipelines can be used."